kie-soup-project-datamodel/kie-soup-project-datamodel-commons/src/main/java/org/kie/soup/project/datamodel/commons/oracle/PackageDataModelOracleImpl.java

```diff
@@ -83,7 +83,7 @@ public void addExtensions(Map<ExtensionKind<?>, List<?>> packageElements) {
         packageElements
                 .forEach((k, v) -> {
                     @SuppressWarnings("rawtypes")
-                    List list = packageElements.computeIfAbsent(k, k1 -> new ArrayList<>());
+                    List list = this.packageElements.computeIfAbsent(k, k1 -> new ArrayList<>());
```

Comment:
I think worth to cover with simple unit test. Let me know what do you think @manstis.

Reply:
@jomarko Unit Test added in new commit.

Reply:
Thanks, looks good. Will approve the ensemble when tried manually.